linkstack/umbrel-app.yml

```
@@ -0,0 +1,49 @@
+manifestVersion: 1
+id: linkstack
+name: LinkStack
+tagline: A highly customizable link sharing platform with an intuitive, easy to use user interface
+icon: https://i.imgur.com/1uzSQWG.png
+category: files
+version: "4.2.3"
+port: 8190
+description: >-
+  LinkStack is a unique platform that provides an efficient solution for managing and sharing links online. Our platform offers a website similar to Linktree, which allows users to overcome the limitation of only being able to add one link on social media platforms.
+
+  With LinkStack, users can easily link to their own custom page and provide their followers with access to all the links they need in one convenient location. What sets LinkStack apart from other link management platforms is its flexibility, which allows users to host their links on their own web server or web hosting provider. This provides users with complete control over their online presence and ensures that their links are easily accessible.
+
+
+  Additionally, LinkStack allows other users to register and create their own links, making it an ideal solution for businesses and organizations looking to manage multiple links. With our user-friendly Admin Panel, managing and accessing other users links is easy.
+
+
+  Why choose LinkStack?
+
+
+  💡 Feature rich: LinkStack offers more features than any other comparable solution out there.
+
+
+  ⚙️ Easy to set up: LinkStack is user-friendly and easier to set up compared to similar web applications.
+
+
+  👐 Open Source: LinkStack is free to use, distribute, and modify. The source code is available on GitHub (https://github.com/LinkStackOrg).
+
+
+
+  ☁️ It is recommended to use LinkStack together with the Cloudflare app from the Umbrel App Store, paired with your own custom domain.
+developer: Julian Prieber
+website: https://linkstack.org/
+submitter: dennysubke
+submission: https://github.com/getumbrel/umbrel-apps/pull/1537
+repo: https://github.com/linkstackorg/linkstack
+support: https://github.com/LinkStackOrg/LinkStack/issues
+gallery:
+  - https://i.imgur.com/hgYPsEe.png
+  - https://i.imgur.com/xGtBotr.png
+  - https://i.imgur.com/gruMpsQ.png
+  - https://i.imgur.com/2ZENf0M.png
+  - https://i.imgur.com/ItJd5X6.png
+releaseNotes: >-
+
```

Can be shortened to this:
```suggestion
releaseNotes: ""
```